CHANGELOG.md

```diff
@@ -30,15 +30,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - Cursors are now inverted when their fixed color is similar to the cell's background
-- Use working directory of active process instead of shell for SpawnNewInstance action
 - Fallback to normal underline for unsupported underline types in `CSI 4 : ? m` escapes
 - The user's background color is now used as the foreground for the render timer
 - Use yellow/red from the config for error and warning messages instead of fixed colors
 - Existing CLI parameters are now passed to instances spawned using `SpawnNewInstance`
 - Wayland's Client side decorations now use the search bar colors
 - Reduce memory usage by up to at least 30% with a full scrollback buffer
 - The number of zerowidth characters per cell is no longer limited to 5
-- `SpawnNewInstance` is now using working directory of active process on macOS
+- `SpawnNewInstance` is now using working directory of the terminal foreground process on macOS/Linux
```

Comment:
Adding the /Linux to the end made things go beyond our 100 character limit per line by 1.

It's also not entirely correct, since it applies to BSD too.